src/conditionalOperators/EndsWith.test.ts

```diff
@@ -0,0 +1,14 @@
+import EndsWith from './EndsWith';
+
+describe('EndsWith operator', () => {
+  const operator = new EndsWith();
+  test('Output should be true', () => {
```

Comment:
should ends with

Reply:
je ne vais pas te le répéter pour tous, je te laisse corriger/améliorer le wording des tests